packages/x-components/src/__stubs__/results-stubs.factory.ts

```diff
@@ -52,16 +52,16 @@ export function createResultStub(name: string, result?: Partial<Result>): Result
 }
 
 /**
- * Function to create a {@link @empathyco/x-types#ResultTagging | result tagging} mocked object
+ * Function to create a {@link @empathyco/x-types#TaggingInfo | result tagging} mocked object
```

Comment:
This is `Tagging` instead of `TaggingInfo` right?

Reply:
Yep, my bad.